src/docker/era_5g_object_detection_standalone/Dockerfile.gpu

```diff
@@ -0,0 +1,58 @@
+FROM but5gera/netapp_base_gstreamer:0.1.1
+
+FROM but5gera/netapp_base_mmcv_gpu:0.1.3
+
+RUN mkdir -p /root/opencv
+
+COPY --from=0 /root/opencv/*.whl /root/opencv/
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=Europe/Prague
+
+RUN apt-get update \
+    && apt-get install -y \
+	libgstreamer1.0-0 \
+	gstreamer1.0-plugins-base \
+	gstreamer1.0-plugins-good \
+	gstreamer1.0-plugins-bad \
+	gstreamer1.0-plugins-ugly \
+	gstreamer1.0-libav \
+	gstreamer1.0-tools \
+	libgstreamer1.0-dev \
+	libgstreamer-plugins-base1.0-dev
+
+RUN cd /root/opencv \
+	&& pip3 install *.whl
+
+RUN wget -c https://download.openmmlab.com/mmdetection/v2.0/yolo/yolov3_mobilenetv2_320_300e_coco/yolov3_mobilenetv2_320_300e_coco_20210719_215349-d18dff72.pth -O "/root/mmdetection/configs/yolo/yolov3_mobilenetv2_320_300e_coco_20210719_215349-d18dff72.pth"
+
+
+
+
+ENTRYPOINT ["/root/start.sh"]
+
+RUN pip3 install simple-websocket
```

Comment:
I would suggest pinning versions everywhere.

Reply:
Yes. Moreover, this particular pip install could be removed as the simple-websocket is specified in the requirements file.

Reply:
Removed